package.json

```diff
@@ -6,7 +6,7 @@
   "types": "index.d.ts",
   "scripts": {
     "autod": "autod",
-    "lint": "eslint lib test *.js",
+    "lint": "eslint --fix lib test *.js",
```

Comment:
Don’t use this flag, it will always be success in ci.

Reply:
This mocks from `egg-security`, where we have this fix to help writers to make codes more beautiful.

Reply:
You can put it in test script, but here is wrong, ci will always be success even if it has style problem.